tests/external/db/test_cur_accounts_db.py

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""Test the CURAccountsDB utility object."""
+
+from masu.external.db.cur_accounts_db import CURAccountsDB
+from tests import MasuTestCase
+
+
+class CURAccountsDBTest(MasuTestCase):
+    """Test Cases for the CURAccountsDB object."""
+
+    def setUp(self):
+        pass
+
+    def test_get_accounts_from_source(self):
+        '''Test to get all accounts'''
```

Comment:
Nit, but can we be consistent with quotes?
``'''Test to get all accounts'''`` --> ``"""Test to get all accounts"""``